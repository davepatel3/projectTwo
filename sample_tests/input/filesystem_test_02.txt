6508
create_directory
kSr-f5
0
create_file
11
FDeU_15yXhNopj43hhz
1
create_directory
wpzUUNskceOI7dB
0
get_file_size_path
/kSr-f5/FDeU_15yXhNopj43hhz
exist
3
print_layout
create_file
32
TwrfUqqd8j_AuxfXuhy
1
create_file
9
jocIyo-
3
create_directory_root
M_S4-OQh-s8qRK
create_file
18
 lVBNmP57Jf
3
create_directory
TFYBjq7Kvb
3
create_directory
Swove3mSsR
1
create_file
22
9E_zQ bStXoDYR0onB4x
0
create_file
238
f6szUYB
9
create_directory
ry4bdXNtH5W
0
create_directory
xgtTiw
9
get_available_size
get_absolute_path
3
get_absolute_path
5
get_available_size
get_absolute_path
11
exist
11
get_available_size
get_file_size
2
create_file
14
2tLJuN
12
create_file
5
jmhdeAiZGgj1E
12
get_available_size
create_directory
kmV5Y6ayRwAdx7k_K5g
0
create_file
188
E P1fQ2THI-bqZWUzHV2
13
get_file_size
14
create_file
68
guD9o6Rl4u
8
create_directory
7XO8c8 BCey Hv6HS
8
create_file
29
vj1lUBmmW4oyGLsT6VSh
9
create_file
77
uiBOe0BLra0Z1
9
get_file_size_path
/wpzUUNskceOI7dB/ lVBNmP57Jf
create_directory
SsU1op4OC
16
get_file_size_path
/kSr-f5/Swove3mSsR/uiBOe0BLra0Z1
create_directory
BfQTvif
8
create_directory
jtviGs0kGy
9
create_file
223
f3EZyw-8
9
get_available_size
create_directory
2hv3NU3dY lBI6LTkaWZ
22
get_absolute_path
12
create_directory
VydIXg2Hn
3
create_file
106
Aeb4n
19
exist
19
create_file
44
z28PARky
8
create_directory
w-4O
8
get_available_size
print_layout
get_available_size
quit
