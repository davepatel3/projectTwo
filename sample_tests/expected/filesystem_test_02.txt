create_directory
1
create_file
2
create_directory
3
get_file_size_path
11
exist
1
print_layout
[D]kSr-f5
	[F]FDeU_15yXhNopj43hhz (size = 11)
[D]wpzUUNskceOI7dB

create_file
4
create_file
5
create_directory_root
6
create_file
7
create_directory
8
create_directory
9
create_file
10
create_file
11
create_directory
12
create_directory
13
get_available_size
6178
get_absolute_path
/wpzUUNskceOI7dB
get_absolute_path
/wpzUUNskceOI7dB/jocIyo-
get_available_size
6178
get_absolute_path
/kSr-f5/Swove3mSsR/f6szUYB
exist
1
get_available_size
6178
get_file_size
11
create_file
14
create_file
15
get_available_size
6159
create_directory
16
create_file
17
get_file_size
14
create_file
18
create_directory
19
create_file
20
create_file
21
get_file_size_path
18
create_directory
22
get_file_size_path
77
create_directory
23
create_directory
24
create_file
25
get_available_size
5574
create_directory
26
get_absolute_path
/ry4bdXNtH5W
create_directory
27
create_file
28
exist
1
create_file
29
create_directory
30
get_available_size
5424
print_layout
[D]kSr-f5
	[F]FDeU_15yXhNopj43hhz (size = 11)
	[F]TwrfUqqd8j_AuxfXuhy (size = 32)
	[D]Swove3mSsR
		[F]f6szUYB (size = 238)
		[D]xgtTiw
			[F]E P1fQ2THI-bqZWUzHV2 (size = 188)
		[F]vj1lUBmmW4oyGLsT6VSh (size = 29)
		[F]uiBOe0BLra0Z1 (size = 77)
		[D]jtviGs0kGy
		[F]f3EZyw-8 (size = 223)
[D]wpzUUNskceOI7dB
	[F]jocIyo- (size = 9)
	[F] lVBNmP57Jf (size = 18)
	[D]TFYBjq7Kvb
		[F]guD9o6Rl4u (size = 68)
		[D]7XO8c8 BCey Hv6HS
			[F]Aeb4n (size = 106)
		[D]BfQTvif
		[F]z28PARky (size = 44)
		[D]w-4O
	[D]VydIXg2Hn
[D]M_S4-OQh-s8qRK
[F]9E_zQ bStXoDYR0onB4x (size = 22)
[D]ry4bdXNtH5W
	[F]2tLJuN (size = 14)
	[F]jmhdeAiZGgj1E (size = 5)
[D]kmV5Y6ayRwAdx7k_K5g
	[D]SsU1op4OC
		[D]2hv3NU3dY lBI6LTkaWZ

get_available_size
5424
quit
